### DISCUSSION TITLE
User interface:  Hz or MHz

### DISCUSSION THREAD

**Discussion by john-** [john-/ham2mon#20] [Q&A]
Ham2mon command line options requires user input as Hz (hertz).  The GUI displays in MHz (megahertz).  The lockout and priority files in this fork are MHz and Hz respectively.  The recently added channel log changes use MHz.

For the user interface (command line, curses interface, channel log), is there any reason these shouldn't exclusively use MHz?  I find this more natural to work with but not sure if there are use cases that make better sense to have Hz.

Of course one reason not to change is that this would be user impacting.  I am asking more for technical reasons but any feedback on user impact would be appreciated.

Internal processing is done in Hz and I am not asking about that use.

**Comment by lordmorgul** [comment]
Consolidating all inputs and displays to MHz works fine for this program.  Even HF listening only really needs to have 5 digits right of MHz at the most, as in 3.59055.
GMRS/2m/70cm in the US uses 462.6125 format at most.

**Reply by john-** [reply]
Good information...thanks.

**Comment by john-** [comment]
This is now merged into master branch.  On the command line and in the configuration file frequencies must be specified in Mhz and not  Hz.
